docs/releases/development.rst

```diff
@@ -2,4 +2,5 @@
 Next release (in development)
 =============================
 
-* ...
+* Fix datasets hash_key generation when geometry encoding
+  is missing a dtype (:pr:`166`).
```

Comment:
Please include a reference to both the relevant issue and the relevant PR:
```suggestion
  is missing a dtype (:issue:`166`, :pr:`167`).
```

Reply:
Done.